gobblin-data-management/src/main/java/gobblin/data/management/copy/replication/ConfigBasedDatasetsFinder.java

```diff
@@ -70,53 +57,50 @@
  */
 
 @Slf4j
-public class ConfigBasedDatasetsFinder implements DatasetsFinder<CopyableDatasetBase> {
+public abstract class ConfigBasedDatasetsFinder implements DatasetsFinder {
 
-  // specify the whitelist tag in the config store used by data replication
-  // the datasets which import this tag will be processed by data replication
-  public static final String GOBBLIN_REPLICATION_CONFIG_STORE_WHITELIST_TAG =
+  // specify the whitelist tag in the config store used by data replication or data retention
+  // the datasets which import this tag will be processed by data replication or data retention
+  public static final String GOBBLIN_CONFIG_STORE_WHITELIST_TAG =
       CopyConfiguration.COPY_PREFIX + ".whitelist.tag";
 
-  // specify the blacklist tags in the config store used by data replication
-  // the datasets which import these tags will NOT be processed by data replication
+  // specify the blacklist tags in the config store used by data replication or data retention
+  // the datasets which import these tags will NOT be processed by data replication or data retention
   // and blacklist override the whitelist
-  public static final String GOBBLIN_REPLICATION_CONFIG_STORE_BLACKLIST_TAGS =
+  public static final String GOBBLIN_CONFIG_STORE_BLACKLIST_TAGS =
       CopyConfiguration.COPY_PREFIX + ".blacklist.tags";
 
-  // specify the common root for all the datasets which will be processed by data replication
-  public static final String GOBBLIN_REPLICATION_CONFIG_STORE_DATASET_COMMON_ROOT =
-      CopyConfiguration.COPY_PREFIX + ".dataset.common.root";
+  // specify the common root for all the datasets which will be processed by data replication/data retention
+  public static final String GOBBLIN_CONFIG_STORE_DATASET_COMMON_ROOT = "dataset.common.root";
```

Comment:
That is intentional just because I haven't come up with a good name for the prefix since it is no longer only for the copy. Will add something else.